src/index.ejs

```diff
@@ -12,6 +12,16 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <% if(__TARGET__ === 'mobile') { %>
   <script src="cordova.js" defer></script>
+  <% } else { %>
+  {{.CozyClientJS}}
```

Comment:
That line removes the need to `import cozy from 'cozy-client-js' and may inject cozy in the global scope, does not it?

But it will be replaced by a script with a local file, we must ship our mobile app with every local files needed.

We, mobile app developers, want to be the closer we can to front developers stack.